org.ferdium.Ferdium.yaml

```diff
@@ -0,0 +1,82 @@
+app-id: org.ferdium.Ferdium
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: ferdium
+separate-locales: false
+rename-desktop-file: ferdium.desktop
+rename-icon: ferdium
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --socket=pulseaudio
+  - --share=network
+  - --device=all
+  - --filesystem=home:ro
+  - --filesystem=xdg-download
+  - --filesystem=/run/.heim_org.h5l.kcm-socket
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --own-name=org.kde.*
+modules:
+  - name: krb5
+    subdir: src
+    config-opts:
+      - --disable-static
+      - --disable-rpath
+    sources:
+      - type: archive
+        url: https://kerberos.org/dist/krb5/1.19/krb5-1.19.3.tar.gz
+        sha256: 56d04863cfddc9d9eb7af17556e043e3537d41c6e545610778676cf551b9dcd0
+        x-checker-data:
+          type: anitya
+          project-id: 13287
+          stable-only: true
+          url-template: https://kerberos.org/dist/krb5/$major.$minor/krb5-$major.$minor.$patch.tar.gz
+  - name: krb5-config
+    buildsystem: simple
+    sources:
+      - type: file
+        path: krb5.conf
+    build-commands:
+      - mkdir -p /app/etc
+      - install -m 644 krb5.conf /app/etc/krb5.conf
+  - name: ferdium-desktop
+    buildsystem: simple
+    build-commands:
+      - ar x ferdium_*.deb
+      - rm -f ferdium_*.deb
+      - tar xf data.tar.xz
+      - rm -f control.tar.gz data.tar.xz debian-binary
+      - cp -r usr/* opt/* /app
+      - chmod -R a-s,go+rX,go-w /app/Ferdium
+      - rm -r /app/share/icons/hicolor/1024x1024
+      - desktop-file-edit --set-key=Exec --set-value='ferdium %U' /app/share/applications/ferdium.desktop
+      - install ferdium.sh /app/bin/ferdium
+      - install -Dm644 org.ferdium.Ferdium.metainfo.xml /app/share/metainfo/org.ferdium.Ferdium.metainfo.xml
+    sources:
+      - type: file
+        url: https://github.com/ferdium/ferdium-app/releases/download/v6.0.0-beta.1/ferdium_6.0.0-beta.1_amd64.deb
```

Comment:
how about adding aarch64 support?

<img width="374" alt="image" src="https://user-images.githubusercontent.com/114441/170702686-d1efeb22-483e-4845-8d42-f4bdab76a240.png">

Reply:
Sounds like a good idea. Will check on that later today.